### ISSUE TITLE
Package creation: missing option to drop version-folder from build-path

### ISSUE THREAD

**Issue by hgh-esn** [elkuku/EasyCreator#49]
When creating a package, EC generates a "... build-path\x.y.z"
This- for me -  is unnecessary, when the filename also contains the x.y.z information, because all files can be uniquely collected in the given build-path.

It will be great, to have an option to drop the x.y.z-folder in the complete build-path.

greetings hgh

**Comment by elkuku** [comment]
I think this is a valid request :wink: 
I thought about completely removing this "feature", as I also don't really find it useful anymore, but I made it optional instead - disabled by default.
This feature will be available in the next version - you may download and test https://github.com/elkuku/EasyCreator/archive/master.zip in the meantime and report and failures please :wink:

**Comment by hgh-esn** [comment]
I installed it in my test-environment.
But there are still some little bugs.

First of all, the build-path now contains no version-folder by default. The generated package can be found there. 

But when going to the build-function there the version-folder is still to see in the displayed build-path.
Also there is no change,, when I set/unset the build-path option in the displayed options-area on that site. 
Also the version-placeholders for the filename are not to be seen.

![2017-10-12_19h45_31](https://user-images.githubusercontent.com/5299726/31513155-aece85cc-af8d-11e7-8303-fe199a10729e.png)

**Comment by hgh-esn** [comment]
reopen because of issues.

**Comment by elkuku** [comment]
OK, I think some of the bugs have been fixed :wink: 

For the custom file name thing: Please go to the "global EC config" and just hit "save". Then go to your projects config and hit the "Reset to defaults" button there.
This 'glitch' is due to the fact that the default config is not been written to the database on install - yet...

**Comment by hgh-esn** [comment]
Hi elkuku,
thanks to you, but where can I find a neu testversion to install?

**Comment by elkuku** [comment]
Right here: https://github.com/elkuku/EasyCreator/archive/master.zip :wink:

**Comment by hgh-esn** [comment]
Hi ,
I now just installed the new version.

It looks quite better now. Also the german language is now active! 

But there are still bugs to work on.

1. The custom file names will never being saved. Even if I worked like your explanation.
2. When building a package, there is no generated output-file. Not in the user defined path and also not in the default build-path.

greetings from germany ... hgh
@elkuku
